tests/js/sentry-test/performance/utils.ts

```diff
@@ -0,0 +1,169 @@
+import {RawSpanType} from 'sentry/components/events/interfaces/spans/types';
+import {EntryType, EventOrGroupType, EventTransaction} from 'sentry/types';
+
+import {PerformanceDetectorData} from './../../../../static/app/types/event';
+
+export enum ProblemSpan {
+  PARENT = 'parent',
+  OFFENDER = 'offender',
+}
+
+export const EXAMPLE_TRANSACTION_TITLE = '/api/0/transaction-test-endpoint/';
+
+type AddSpanOpts = {
+  endTimestamp: number;
+  startTimestamp: number;
```

Comment:
Should we have defaults for the timestamp? Like in the case I don't case about them for my particular test?

Reply:
I'm going to skip this actually, I think implementing this would be more effort than it's worth. I think testers should always give timestamps, otherwise there could be unexpected behaviour if the builder determines the timestamps on its own.